tests/test_tilelayer.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+
+from .base import TileLayerFactory
+
+pytestmark = pytest.mark.django_db
+
+
+def test_tilelayer_json():
+    tilelayer = TileLayerFactory(attribution='Attribution', maxZoom=19,
+                                 minZoom=0, name='Name', rank=1, tms=True,
+                                 url_template='http://{s}.x.fr/{z}/{x}/{y}')
+    assert tilelayer.json == {
+        'attribution': 'Attribution',
+        'id': tilelayer.id,
+        'maxZoom': 19,
+        'minZoom': 0,
+        'name': 'Name',
+        'rank': 1,
+        'tms': True,
```

Comment:
Humm maybe the name is bad, but the `json` method return the object ready *for json* serialization, but the serialization itself is made at the end of the process, when all the `json` call have been joined in a `map_setting` dict.